liquibase-standard/src/main/java/liquibase/executor/jvm/StatementCreatorUtils.java

```diff
@@ -80,7 +82,7 @@ private static void setParameterValueInternal(
                     useSetObject = (ps.getConnection().getMetaData().getDatabaseProductName().contains("Informix"));
                 }
                 catch (Throwable ex) {
-//                    logger.debug("Could not check database product name", ex);
+                    Scope.getCurrentScope().getLog(StatementCreatorUtils.class).fine("Could not check database product name", ex);
```

Comment:
Something very minor, but I would keep the same log level (debug) here or at least change it as a `warning`.

Reply:
@MalloD12 But "fine" is the same as "debug". :)

Look, this is javadoc of `liquibase.logging.Logger`:

<img width="726" alt="image" src="https://github.com/user-attachments/assets/a9caacbc-d310-45a4-bb12-83411abeeb7f">

Reply:
Yeah, according to that class, it's correct what you have done. But I'm not sure why `debug` methods are deprecated here. I'll check with the team. 

Thanks,
Daniel.